v3/setup/capcom.sh

```diff
@@ -7,26 +7,24 @@ if [ "${NODE_ROLE}" != "control" ]  && [ "${NODE_ROLE}" != "it-hybrid" ]; then
     exit 0
 fi
 
-if [ "$(etcdctl get capcom-bootstrapped)" == "true" ]; then
-    exit 0
+etcdctl get /capcom/config/proxy
```

Comment:
That works - just rely on it being set or not to determine if it was bootstrapped. As long as this file runs _before_ the etcd-seed script. If not, the user could set this value in the config file and then this script would think all the values were already populated.